BGL/include/CGAL/boost/graph/named_params_helper.h

```diff
@@ -435,6 +435,30 @@ CGAL_DEF_GET_INITIALIZED_INDEX_MAP(face, typename boost::graph_traits<Graph>::fa
         > ::type  type;
     };
 
+    template<typename PointRange, typename NamedParameters>
+    class GetScalarMap
+    {
+      struct DummyScalarMap
```

Comment:
You can probably use `Constant_property_map` (see in `property_map.h` in the Property map package), provided you initialize it with the 1 value in the PSP code.

Reply:
Cool. This is what I was searching for. I will change. Thanks!